packages/web-react/src/components/RadioField/__tests__/RadioField.test.tsx

```diff
@@ -36,4 +36,32 @@ describe('RadioField', () => {
     const element = dom.container.querySelector('input') as HTMLElement;
     expect(element).toHaveClass('RadioField__input');
   });
+
+  it('should have helper text', () => {
+    const dom = render(<RadioField id="radiofield" label="Label" helperText="text" />);
+
+    const element = dom.container.querySelector('.RadioField__helperText') as HTMLElement;
+    expect(element.textContent).toBe('text');
+  });
+
+  it('should have danger classname', () => {
```

Comment:
When having validation state props test, it can be used also here.

Reply:
done